tests/testEmPsfApprox.py

```diff
@@ -0,0 +1,208 @@
+# LSST Data Management System
+# Copyright 2008-2015 AURA/LSST
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+import re, os, sys
+import glob
+import math
+import numpy as np
+import unittest
+import itertools
+import lsst.pex.exceptions as pexExceptions
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.utils.tests as utilsTests
+import lsst.afw.detection as afwDetection
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+import lsst.afw.geom.ellipses as afwEll
+import lsst.afw.coord as afwCoord
+import lsst.afw.display.ds9 as ds9
+import lsst.meas.algorithms as measAlg
+import lsst.meas.base as measBase
+import lsst.meas.base.tests
+import lsst.meas.extensions.ngmix.emPsfApprox
+
+#   Create an array of size x size containing a 2D circular Gaussian of size sigma.  Normalized to 1.0
+def makeGaussianArray(size, sigma, xc=None, yc=None):
+    if xc == None:
+        xc = (size-1)/2
+    if yc == None:
+        yc = (size-1)/2
+    image = afwImage.ImageD(afwGeom.Box2I(afwGeom.Point2I(0,0), afwGeom.Point2I(size,size)))
+    array = np.ndarray(shape = (size, size), dtype = np.float64)
+    for yi, yv in enumerate(xrange(0, size)):
+        for xi, xv in enumerate(xrange(0, size)):
+            array[yi, xi] = np.exp(-0.5*((xv - xc)**2 + (yv - yc)**2)/sigma**2)
```

Comment:
I took this code from somewhere else in the DM stack (afw/tests)  I didn't look at it that closely.  Do you know of a service routine so that I don't need to recreate this code?